src/App.js

```diff
@@ -33,12 +34,14 @@ const App = ({ loadTechnologies }) => {
 
     return (
         <PermissionContext.Provider value={ { isOrgAdmin } }>
-            <Switch>
-                <Route exact path={ routes.landing } component={ Landing } />
-                <Route exact path={ routes.landingBeta } component={ Landing } />
-                <Route exact path={ routes.maintenance } component={ Maintenance } />
-                <Route path="*" component={ NotFound } />
-            </Switch>
+            <Suspense fallback={ <Fragment></Fragment> }>
```

Comment:
Can you use `Spinner` component as fallback? With just `Fragment` it'll show empty page while the chunk is being loaded.

Reply:
I think it will be a bit disruptive to just flash the spinner for a couple of ms, especially if the user almost never switches between these views intentionally.

Reply:
Rahter flash the spinner to indicate something is going instead of blank page.